components/dashboard/src/workspaces/WorkspaceEntry.tsx

```diff
@@ -46,54 +43,29 @@ export const WorkspaceEntry: FunctionComponent<Props> = ({ info, shortVersion })
     );
 
     return (
-        <Item className="whitespace-nowrap py-6 px-6" solid={menuActive}>
+        <Item className="whitespace-nowrap py-2 px-2" solid={menuActive}>
             <ItemFieldIcon>
                 <WorkspaceStatusIndicator instance={info?.latestInstance} />
             </ItemFieldIcon>
-            <ItemField className="w-3/12 flex flex-col my-auto">
+            <ItemField className="w-3/12 my-auto">
                 <a href={startUrl}>
-                    <div className="font-medium text-gray-800 dark:text-gray-200 truncate hover:text-blue-600 dark:hover:text-blue-400">
-                        {workspace.id}
+                    <span className="font-medium text-gray-800 dark:text-gray-200 truncate hover:text-blue-600 dark:hover:text-blue-400 max-w-xs overflow-hidden whitespace-nowrap truncate">
```

Comment:
**thought**: The removal of most of the information in this component would make this also quite responsive. Ideally, we could also enforce a max width of the branch name and truncate it on smaller viewports, but should be fine as is for now.